tests/test_modules/test_connections.py

```diff
@@ -332,6 +330,74 @@ def test_connect_finalize_correctness(self, load_rod_with_connects):
             assert sconnect is None
             assert type(connect) is connect_cls
 
-    def test_connect_call_on_systems(self):
-        # TODO Finish after the architecture is complete
-        pass
+    @pytest.fixture
+    def load_rod_with_connects_and_indices(self, load_system_with_connects):
+        scwcai = load_system_with_connects
```

Comment:
what is this holy word? scwai?

Reply:
spell out things, make intent clear

Reply:
Whoever wrote the test code before me used scwc for SystemCollectionWithConnections, I just followed along and used scwcai for SystemCollectionWithConnectionsAndIndices. Should I just change just this or also the scwc for SystemCollectionWithConnections?

Reply:
maybe spell it first time, and then alias a variable name for instance,
```
scwcai = system_collection_with_connections_and_indices
```